src/include/distributed/multi_physical_planner.h

```diff
@@ -328,6 +328,11 @@ typedef struct Task
 	 * ExplainTaskList().
 	 */
 	double fetchedExplainAnalyzeExecutionDuration;
+
+	/*
+	 * containsOnlyLocalTable is true if the task contains only postgres table/MV.
+	 */
+	bool containsOnlyLocalTable;
```

Comment:
It's quite cool that modifying local tables works, and it's obvious why that's a special case, but it's not obvious from the code that this flag exists to address that.

Could we change the meaning and naming to something more specific like `isLocalTableModification`?